website/docs/tutorials/managing-access.md

```diff
@@ -123,19 +123,150 @@ const accessByAgent = unstable_getAgentAccessAll(litDatasetWithAcl);
 
 ## Changing who has access
 
-:::caution
-
-This section is still being written.
-
-:::
+To be able to _change_ who has access to a specific Resource, you will need to know a little bit
+more about how Access Control works in Solid:
 
 ### Two types of ACL
 
 A Resource _can_ have an ACL that applies to just that Resource. However, if no such ACL exists, the
-Pod server will fall back to the ACL of its [Container](../glossary#container) — or its Container's
+Pod server will [fall back](../glossary#fallback-acl) to the ACL of its [Container](../glossary#container) — or its Container's
```

Comment:
Just wondering - should this be actually up at top where we introduce Access Control in Solid?

## Changing who has access:

To modify someone's access to a Resource, you will need to obtain its Resource ACL; i.e., 

1. Fetch the Resource using
  `unstable_fetchLitDatasetWithAcl`](../api/modules/_litdataset_#unstabl
  e_fetchlitdatasetwithacl)

2. Retrieve its Resource ACL using `unstable_getResourceACL`](../api/modules/_acl_acl_#unstable_getresourceacl).  
    a. If the resource does not have a Resource ACL, the function returns `null`.
...
3. Set the ACL for the agent.

4. Save the ACL for the agent using `unstable_saveAclFor(litDatasetWithAcl, updatedAcl)`](../api/modules/_acl_acl_#unstable_saveaclfor]`.
And later on, where we set the agent's default acl, we can just blurb a sentence reminding people that the default applies to children and not the resource.

Reply:
Do you mean the entire section on changing access (i.e. moving "changing access" to before "reading access"), or just the part you quoted? The latter feels weird to me, since this explicitly describes the process of _changing_ access; _reading_ access consists of other steps.

(Or maybe this is related to the other thing I was having trouble with here? Which was that to read access, you don't necessarily need to know about the distinction between the Resource and Fallback ACL yet, even though it's technically still there, and those who read access are probably (?) doing so because they want to modify it later?)